### ISSUE TITLE
Accessibility Roles and Responsibilities Mapping (ARRM)

### ISSUE THREAD

**Issue by mgifford** [w3c/tpac2025-breakouts#2]
### Session description

A discussion about Accessibility Roles and Responsibilities Mapping (ARRM) and its role in helping accessibility evaluation.

[Slack](https://w3ccommunity.slack.com/archives/C07CN8EGAPQ)

### Session goal

Review progress thus far and look to opportunities ahead

### Who can attend

Anyone may attend (Default)

### Instructions for meeting planners (Optional)

My priority is the Web Sustainability Guidelines meetings, so I can't have overlaps with these items [TPAC Schedule](https://www.w3.org/2025/11/TPAC/schedule.html#thursday) (JP Time):

- 9:00am - 12:30pm = Our IG Meeting.
- 13:45pm - 15:00pm = Us + Web Perf WG
- 15:30pm - 16:45pm = Us + A11y Guidelines WG

### IRC channel (Optional)

[`#arrm`](https://webirc.w3.org/?channels=arrm)

### Agenda for the meeting.

- Review ARRM & its history
- Look at the roadmap for a teams based approach
- Consider the Web Sustainability Guidelines approach to highlight roles
- Consider implications for future WCAG releases

### Links to calendar

- [Wednesday, 16:15 - 17:15](https://www.w3.org/events/meetings/76eeff8c-9750-4d3e-874b-a32293d6af9c/)
